ONSdigital/srm-rh-ui
# Add Welsh translation for cookies page

# Motivation and Context

There was some missing translation on the Welsh cookies page
# What has changed



Translation added
# How to test?



make build and start in your own env
Check the page and ensure the text matches the document attached to the Jira card
Run ATs
# Links



https://jira.ons.gov.uk/browse/SDCSRM-383
# Screenshots (if appropriate)
